src/main/java/org/mitre/synthea/world/concepts/HealthRecord.java

```diff
@@ -176,6 +177,41 @@ public CarePlan(long time, String type) {
     }
   }
 
+  public class ImagingStudy extends Entry {
+    public List<Series> series;
+
+    public ImagingStudy(long time, String type) {
+      super(time, type);
+      this.series = new ArrayList<Series>();
+    }
+
+    /**
+     * ImagingStudy.Series represents a series of images that were taken of
+     * a specific part of the body.
+     */
+    public class Series {
+      /* A SNOMED-CT body structures code */
+      @SerializedName("body_site")
```

Comment:
Are these `@SerializedName`s necessary? The Gson setting is specified that field names get changed from camelCase variables to lower case with underscores in the serialized json.  https://github.com/synthetichealth/synthea/blob/master/src/main/java/org/mitre/synthea/helpers/Utilities.java#L223  True it's more explicit but I don't think it's required in this case.


Also, re: the comments here, please make those javadocs.

Reply:
Good catch on the `@SerializedName`, it is indeed unnecessary and I'll remove it. That was a leftover from my experimentation with the deserialization of ImagingStudy states.